### PR TITLE
fix: children field for button

### PR DESCRIPTION
**Pull request by chungnguyenhoang**
## Check List
- [ ] Your code builds clean without any errors or warnings.
- [ ] Check coding convention.
- [ ] Updated Storybook components and stories to reflect any changes made to the UI.
- [ ] Test covers all edge cases and coverage is greater than or equal to 80%.

## Proof of Completeness
Screenshot changes introduced" width="250" />
OR
Video link

### CODE REVIEW

File: src/components/Button/Button.tsx [hunk 1]
```diff
@@ -91,9 +92,13 @@ const Button: React.FC<ButtonProps> = ({
       style={[{height: ButtonTheme.height}, StyleSheet.flatten(style)]}
       {...props}>
       {!!leftIcon && leftIcon}
-      <Label {...textProps} style={textStyle} color={textColor ?? ButtonTheme.textColor}>
-        {text}
-      </Label>
+      {children ? (
```
Comment (ledutu2) [inline src/components/Button/Button.tsx:95]: I think you should check typeof children. if children is string:
it will return ``<Label>{children}</Label>``
else:
return ``{children}``
and remove text props

File: src/components/Button/Button.tsx [hunk 2]
```diff
@@ -12,8 +12,8 @@ import type {ITheme} from '../../theme'
 import {useTheme} from '../../hooks'
 
 export type ButtonProps = {
-  onPress: () => void
-  text: string
+  onPress?: () => void
```
Comment (baonguyenhsts) [inline src/components/Button/Button.tsx:15]:
```suggestion
 onPress?: (e: GestureResponderEvent) => void
```

**Review by baonguyenhsts — CHANGES_REQUESTED** [review]

**Review by baonguyenhsts — APPROVED** [review]
